Sources/Einstein/Shared/Image/ImageCache.swift

```diff
@@ -0,0 +1,57 @@
+//
+//  ImageCache.swift
+//  
+//
+//  Created by Richard Blanchard on 6/23/19.
+//
+
+import Combine
+import Foundation
+import UIKit
+
+public class ImageCache {
+    
+    public enum Error: Swift.Error {
+        case couldNotConvertData(URL)
+        case general(Swift.Error )
+    }
+    
+    private var imageMap: [URL: UIImage] = [:]
```

Comment:
Should we create a ticket for the future to store the ImageCache on the local disk so it can persist between application launches?